dojo/tools/nexpose/parser.py

```diff
@@ -229,7 +229,11 @@ def get_items(self, tree, vulns, test):
 
                     find = self.findings(dupe_key, dupes, test, vuln)
 
-                    endpoint = Endpoint(host=host['name'], port=service['port'], protocol=service['name'].lower())
+                    endpoint = Endpoint(
+                        protocol=service['name'].lower(),
```

Comment:
Could you switch this to `Endpoint.from_uri()` ?

Reply:
Nexpose store IP/hostname, port, service in ["raw" format](https://github.com/DefectDojo/django-DefectDojo/blob/580d014a26cb6d32346646f52016668a2e6aff46/dojo/unittests/scans/nexpose/many_vulns.xml#L41-L43), not in URL.
I suppose it is better to use these value than create URL and parse it again

Reply:
ok, let's keep it like that for now.